src/runtime_src/core/pcie/emulation/cpu_em/generic_pcie_hal2/shim.cxx

```diff
@@ -80,6 +80,8 @@ namespace xclcpuemhal2 {
     mCloseAll = false;
     bUnified = _unified;
     bXPR = _xpr;
+    mCore = nullptr;
+    mSWSch = nullptr;
```

Comment:
Why are these and other data members not initialized in the initializer-list ?

Reply:
Coverity is showing these variables are being used without initializing them. Hence, added them here.

Reply:
You can move them to the constructor member initializer list; that is preferable.